lib/gstreamer/va/decoder.py

```diff
@@ -14,12 +14,17 @@
 
 class Decoder(GstDecoder):
   format = property(lambda s: mapformatu(super().format))
+  hwdevice = get_media().render_device.split('/')[-1]
```

Comment:
This call gets invoked when this file is imported.  This will cause problems since the media plugin instance may not be initialized, yet.  For example:
```shell
$ smoke list test/gst-va/
Could not load tests (Could not load 'test/gst-va/decode/av1.py' (/home/uaeoff/Work/workspace/media/src/vaapi-fits/lib/gstreamer/va/decoder.py:17 - 'MediaPlugin' object has no attribute 'render_device'))
```
Please wrap this call in a property so that it is only evaluated at test runtime just before it is actually used.